systemd/Makefile.am

```diff
@@ -0,0 +1,10 @@
+systemdunitdir=@SYSTEMD_UNIT_DIR@
+
+dist_systemdunit_DATA = \
+	authselect.service \
+	$(NULL)
+
+# Hook this into sysinit.target by default
+install-exec-hook:
+	$(MKDIR_P) $(DESTDIR)/$(systemdunitdir)/sysinit.target.wants
+	$(LN_S) -f ../authselect.service $(DESTDIR)/$(systemdunitdir)/sysinit.target.wants/authselect.service
```

Comment:
Why not define an `[Install]` section in the unit file and make this dynamic? Static enablement can be pretty annoying.

Reply:
Dunno, the system is not really usable without PAM working, hence I doubt we should advertise this as something you can disable.

Reply:
Yeah, but it's possible that you want to generate the config in a different way, for example config mgmt, or not update automatically, etc. When the enablement is hardcoded like that, the only easy way to disable is to mask, which creates other problems.